docker/docker-compose.debug.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.5'
+
+services:
+
+  db:
+    image: nhsd-buying-catalogue-debug-db:latest
```

Comment:
The image in the development docker file uses slashes. Can we follow that pattern?

Reply:
👍